README.md

```diff
@@ -193,7 +194,7 @@ metadata:
   name: pod-failure-example
   namespace: chaos-testing
 spec:
-  action: pod-failure # the specific chaos action to inject; supported actions: pod-kill/pod-failure
+  action: pod-failure # the specific chaos action to inject; supported actions: pod-kill/pod-failure/container-kill
```

Comment:
Did we need to specify container name?

Reply:
Yes. But the example here the action is pod-failure, so there is no information describing the container name.

Reply:
so. maybe we can remote container-kill here